html/themes/custom/iasc_common_design/templates/views/views-exposed-form.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme override for a views exposed form.
+ *
+ * Available variables:
+ * - form: A render element representing the form.
+ *
+ * @see template_preprocess_views_exposed_form()
+ */
+#}
+{% if q is not empty %}
+  {#
+    This ensures that, if clean URLs are off, the 'q' is added first,
+    as a hidden form element, so that it shows up first in the POST URL.
+  #}
+{{ q }}
+{% endif %}
+
+{{ attach_library('iasc_common_design/cd-button') }}
+{{ attach_library('iasc_common_design/cd-filters') }}
+<div class="cd-filters form--inline clearfix">
+  <button type="button" class="cd-filters_btn cd-button" id="cd-filters-btn" data-toggle="dropdown" id="cd-filter-dropdown-toggle" aria-expanded="false" aria-label="Filter">
+    <span class="cd-filters_btn-text">Filter</span>
+    <svg class="icon icon--arrow-down">
+      <use xlink:href="#arrow-down"></use>
+    </svg>
+  </button>
+  <div class="cd-dropdown cd-filters__form" id="cd-filter-dropdown" aria-labelledby="cd-filter-dropdown-toggle">
```

Comment:
@attiks I've added a template override to the Views exposed forms. It will affect all exposed forms, except the global search, hiding them on page load, and adding a Filter button to toggle.
Like https://interagencystandingcommittee.org/resources
FYI in case it causes issues. I'm not sure if its too generic.